src/routers/api.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy.orm import Session
+
```

Comment:
```suggestion
from fastapi_sqlalchemy import db
```
session은 지금 여기서 필요하진 않을거 같고, 사용하게 될 경우에는 위와 같이 불러서 사용해야 할거 같아